generated/nidigital/session.py

```diff
@@ -1087,6 +1091,8 @@ def _get_error(self):
         Returns:
             error_code (int):
 
+            error_description (str):
```

Comment:
There is a trailing colon because you put the description after the colon. We just don't have a description yet, but we want to keep the colon since that tells sphinx where the name and type information end.